CONTRIBUTING.md

```diff
@@ -0,0 +1,72 @@
+# What this repository is
+
```

Comment:
It makes sense to apply our discussions to Skiko repo as well. Issues appeared only after reading it for the second time, through the Compose prism :)